assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -88,11 +102,78 @@ function ColorPreview({ onChange, hasGradient, hasOpacity, value, label }) {
     hasOpacity,
   ]);
 
+  const [hexInputValue, setHexInputValue] = useState('');
+
+  useEffect(() => setHexInputValue(previewText), [previewText]);
+
+  const colorType = value?.type;
+  const isEditable =
+    !isMultiple &&
+    Boolean(previewText) &&
+    (!colorType || colorType === 'solid');
+
+  const editLabel = __('Edit', 'web-stories');
+  const inputLabel = __('Enter', 'web-stories');
+
+  const buttonProps = {
+    as: 'button',
+    type: 'button', // avoid submitting forms
+    onClick: handleOpenEditing,
+    'aria-label': `${editLabel}: ${label}`,
+  };
+
+  const handleInputChange = useCallback(
+    (evt) => {
+      const raw = evt.target.value.trim();
+      // Strip initial '#' (might very well be pasted in)
+      const val = raw.charAt(0) === '#' ? raw.substr(1) : raw;
```

Comment:
Why not only use `raw.replace('#','')` ?

Reply:
I didn't feel like policing all the things people could paste in here. Anything can be inserted, but in the very special case that it starts with a `#`, I'll remove it.

I don't like changing pasted content when it isn't necessary - it confuses users IMO.

And if you have a `#` somewhere other than first place, you've pasted something weird in anyway.

Reply:
Gotcha! What about `raw.replace(/^\#/, "")`? It will ensure the only if `#` is the first character of the string it should be replaced.

Reply:
Yeah true, that would be simpler.

Reply:
Fixed in d2580391.